descqagen/app_mag_func_test/data/download_data.py

```diff
@@ -0,0 +1,57 @@
+"""
+run sql queries on HSC database
+"""
+
+from __future__ import print_function, division
+import sys
+sys.path.append("..")
+from defaults import PROJECT_DIRECTORY, DATA_DIRECTORY
+
+
+def main():
+    """
+    run scripted SQL download.
+
+    to download the HSC data for this project, in the temrinal window type:
+        $user python download_data.py data
+
+    to download the HSC randoms for this project, in the temrinal window type:
+        $user python download_data.py randoms
+
+    Before running this script, edit the hsc_credentials.txt file in the sql directory.
+    """
+
+    if sys.argv[1] == 'data':
+        sql_file = PROJECT_DIRECTORY + 'sql/hsc_data.sql'
+        savename = DATA_DIRECTORY + 'hsc_data.csv'
```

Comment:
`savename` is not used in the following script. Is this intended?

Reply:
hmmm, I dont see where the filename that is used to save the data is specified...  let me try to reload this into my memory...

Reply:
Just a gentle reminder: @duncandc any luck on solving this minor issue?